tests/console/verify_efi_mok.pm

```diff
@@ -93,13 +94,18 @@ sub check_efi_state {
         record_info "Fallback", "EFI label: $found->{label}";
     }
 
-    # return only if SecureBoot is off or image has booted n Fallback mode
-    return if (get_var('DISABLE_SECUREBOOT', 0) || (!$found->{exec} && !$expected->{exec}));
-
-    my $issuer_regex = qr/Secure\s+Boot\s+CA/;
-    diag("Check presence of signature in shim");
-    assert_script_run("pesign -S -i $expected->{mount}/$found->{exec}");
-    push @errors, 'No openSUSE/SUSE keys found in keyring(/proc/keys)' if script_output('cat /proc/keys') !~ $issuer_regex;
+    if (!get_var('DISABLE_SECUREBOOT', 0) && $found->{exec} && $expected->{exec}) {
+        diag("Check presence of signature in shim");
+        assert_script_run("pesign -S -i $expected->{mount}/$found->{exec}");
+        #check if MokListRT is present in kernel's keyring
+        if (script_output('cat /proc/keys') !~ qr/Secure\s+Boot\s+CA/) {
+            if (is_sle('15-SP3+')) {
```

Comment:
Is this SLE specific? FWICT it affects all aarch64:

```
            if (is_aarch64) {
```

Reply:
Oops, true. Thanks!